invision-app/app/components/movieCard/movieCard.js

```diff
@@ -81,9 +76,13 @@ MovieCard.propTypes = {
     genres: PropTypes.string.isRequired,
     runtime: PropTypes.oneOfType([PropTypes.string.isRequired, PropTypes.number.isRequired])
   }),
-  updateMovie: PropTypes.func.isRequired,
-  //deleteMovie: PropTypes.func.isRequired, 
-  //changeHeaderToDescription: PropTypes.func.isRequired
+  showMovieDetails: PropTypes.func.isRequired
+  
+};
+
+const mapDispatchToProps = {
+  updateMovie
+  
 };
 
-export default MovieCard;
\ No newline at end of file
+export default connect(null,mapDispatchToProps)(MovieCard);
```

Comment:
You do not need **connect** and **mapDispatchToProps** in this component - use Redux hooks. You already do that with the **showDescriptionMovie** function in this component